### PR TITLE
Display Popular Vote Leaderboard

### PR DESCRIPTION
**Pull request by DesSnowy**
**Description**
This pull request supports the usage of a new tab which shows the popular votes leaderboard in the contest voting assessments. Relevant frontend pull request has been opened. 

**Changes To Be Made**

- [X] Added new popular_score field to answers 
- [X] Added new methods which mirrors calculation for relative_score to popular_score
- [X] Added new popularVoteLeaderboard output for voting questions
- [X] Changed testcases to include popularVoteLeaderboard

### CODE REVIEW

File: lib/cadet/assessments/assessments.ex [hunk 1]
```diff
@@ -1372,7 +1466,8 @@ defmodule Cadet.Assessments do
       |> Enum.map(fn ans ->
         if ans.question.type == :voting do
           empty_contest_entries = Map.put(ans.question.question, :contest_entries, [])
-          empty_contest_leaderboard = Map.put(empty_contest_entries, :contest_leaderboard, [])
+          empty_popular_leaderboard = Map.put(empty_contest_entries, :popular_leaderboard, [])
+          empty_contest_leaderboard = Map.put(empty_popular_leaderboard, :contest_leaderboard, [])
```
Comment (RichDom2185) [inline lib/cadet/assessments/assessments.ex:1470]: Are these supposed to be the other way around?

**Review by RichDom2185 — COMMENTED** [review]
Hi, sorry for the delayed review. Thanks for working on this, I just have one comment below:

File: lib/cadet/assessments/assessments.ex (hunk 1 above)
Comment (DesSnowy) [reply lib/cadet/assessments/assessments.ex:1607]: Hi, it is meant to be like this: the result from the previous line will be used in the next line

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**